wire-ios-sync-engine/Source/Use cases/ResolveOneOnOneConversationsUseCase.swift

```diff
@@ -49,6 +49,7 @@ struct ResolveOneOnOneConversationsUseCase: ResolveOneOnOneConversationsUseCaseP
             }
         }
 
+        // Do we need to remove this check?
```

Comment:
it would mean you cleanup the connections more often, on every quick sync.

Reply:
only when mls is enabled, so it won't make an impact, let's keep the check for now